client/a8c-for-agencies/sections/marketplace/checkout/product-info.tsx

```diff
@@ -115,7 +121,28 @@ export default function ProductInfo( {
 			<div className="product-info__text-content">
 				<div className="product-info__header">
 					<label htmlFor={ productTitle } className="product-info__label">
-						{ productTitle }
+						<h3>{ productTitle }</h3>
+						{ vendor &&
+							translate( 'By {{a/}}', {
+								components: {
+									a: (
+										<a
+											href={ vendor.vendorUrl }
+											target="_blank"
+											rel="noopener noreferrer"
+											onClick={ () => {
+												dispatch(
+													recordTracksEvent( 'calypso_marketplace_products_overview_vendor_click', {
+														vendor: vendor.vendorName,
+													} )
+												);
+											} }
+										>
+											{ vendor.vendorName }
+										</a>
+									),
+								},
+							} ) }
```

Comment:
I think we can create a common component and reuse it wherever required, as it is being used in 2 places

`<ProductVendorInfo />` or something similar

Reply:
At first, I had the same thought, but since the title is not something we can share both the Lightbox and Checkout summary due to Typography, we are limited to just a translation string. I decided not to create a component for now.

Reply:
Sorry, I was talking only about the vendor.